scripts/sample_data/sample_traffic.csv

```diff
@@ -0,0 +1,978 @@
+surface_type,collision_type,severity,surface_condition,num_driver_casualties,lon,traffic_control,num_vehicles,num_passenger_casualties,num_pedestrian_casualties,record_date,lat,main_cause
+Asphalt,Hit parked vehicle,Property,Dry,0,0,None,1,0,0,06/18/18,0,Human error
+Concrete,Side swipe,Property,Dry,0,-0.00014008239912971835,Traffic lights,2,0,0,06/18/18,0.00020447453828494324,Human error
```

Comment:
These coordinates don't look right -- they're all clustered around (0,0) (off the west coast of Africa), which makes me think that perhaps rather than _adding_ jitter to the coordinates, each coordinate was _replaced_ with the jitter?

Reply:
Well that's frustrating... I fixed and committed this before I left on vacation, but apparently forgot to push the change to `origin`. This should be fixed there now 😣